code/planning/launch/planning.launch

```diff
@@ -1,4 +1,4 @@
 <!-- TODO: Insert components of component-->
 <launch>
-    
-</launch>
\ No newline at end of file
+
```

Comment:
nothing is beeing launched here propably because of WIP but do we really wanna merge all those following python files into main right now if they are not used and still in progress. Eventually draft PR would be something

Reply:
I changed the launch file to test some sensors from the planning node.
It was only for test purposes. 
Yes I agree with you to change it to a draft PR. It is not finished to test it really good. Also we have to find solution to visualize it in the Simulator (#71). It comes out that there are a lot of things to take care of, when reading the openDrive file. That is why i am still working on it and need a possibility to visualize and debug it in the Simulator.